include/grpcpp/support/channel_arguments.h

```diff
@@ -70,13 +70,7 @@ class ChannelArguments {
   /// the resolver.
   void SetGrpclbFallbackTimeout(int fallback_timeout);
 
-  /// For client channel's, the socket mutator operates on
-  /// "channel" sockets. For server's, the socket mutator operates
-  /// only on "listen" sockets.
-  /// TODO(apolcyn): allow socket mutators to also operate
-  /// on server "channel" sockets, and adjust the socket mutator
-  /// object to be more speficic about which type of socket
-  /// it should operate on.
+  /// Set a mutator for the underlying socket.
   void SetSocketMutator(grpc_socket_mutator* mutator);
```

Comment:
The fact that we added this to the API is a bit strange, since the `grpc_socket_mutator` type itself is not part of the public API.  Maybe we should mark this as experimental while we're at it?